hypothesis-python/RELEASE.rst

```diff
@@ -0,0 +1,4 @@
+RELEASE_TYPE: patch
+
+This patch makes stateful step printing expand the result of a step into
+multiple variables when a MultipleResult is returned (:issue:`2139`).
```

Comment:
We'd love to have an extra line at the end

   Thanks to Joseph Weston for reporting and fixing this bug!

if you're comfortable with that ❤️

Reply:
I added it; see below